src/python/grapl-tests-common/grapl_tests_common/clients/graphql_endpoint_client.py

```diff
@@ -0,0 +1,18 @@
+import os
+import requests
+
+
+class GraphqlEndpointClient:
+    def __init__(self, jwt: str) -> None:
+        hostname = "grapl-graphql-endpoint"
+        port = os.environ["GRAPL_GRAPHQL_PORT"]
+        self.endpoint = f"http://{hostname}:{port}"
```

Comment:
Would it be better to just provide a `GRAPL_GRAPHQL_ENDPOINT` value, and not have to piece it together (also assuming a hard-coded hostname)?

Reply:
eh, i think the host + port is easy enough; + there are some libs that prefer specifying something like `port=` than adding `:${PORT}` to the endpoint url.

i will add a GRAPL_GRAPHQL_HOSTNAME though